app/assets/stylesheets/sign_in.scss

```diff
@@ -0,0 +1,8 @@
+.thin-form {
```

Comment:
This file is called `sign_in` but the styles are for a `.thin-form`. So that developers know what to expect in this file and to let other developers know there is generic styling for thin forms can this be called `thin_form.scss` or could the selector be made to only attach to `sign-in` elements?